message_tagging_service/consumer.py

```diff
@@ -30,22 +29,104 @@
 logger = logging.getLogger(__name__)
 
 
-class MTSConsumer(fedmsg.consumers.FedmsgConsumer):
-    topic = conf.consumer_topics
-    config_key = 'mts-consumer'
+class RHMessage(object):
+    """Representing a message received from rhmsg message bus"""
 
-    def __init__(self, *args, **kwargs):
-        super(MTSConsumer, self).__init__(*args, **kwargs)
+    def __init__(self, raw_msg):
+        self._raw_msg = raw_msg
 
-    def consume(self, msg):
-        logger.debug('Got message: %r', msg)
+    @property
+    def id(self):
+        return self._raw_msg['msg_id']
 
-        try:
-            rule_defs = read_rule_defs()
-        except requests.exceptions.HTTPError:
-            logger.exception('Failed to retrieve rules content.')
+    @property
+    def topic(self):
+        return self._raw_msg['topic']
+
+    @property
+    def body(self):
+        return self._raw_msg['msg']
+
+    def __repr__(self):
+        return "{}(id={}, topic={}, body={})".format(
+            self.__class__.__name__, repr(self.id), repr(self.topic), repr(self.body)
+        )
+
+
+def consume(msg):
+    """Do the work to tag build if it matches a rule
+
+    :param dict msg: the message got from message bus.
+    """
+    try:
+        rule_defs = read_rule_defs()
+    except requests.exceptions.HTTPError:
+        logger.exception('Failed to retrieve rules content.')
+        return
+    else:
```

Comment:
The `else` block is unnecessary because there's a `return` in the `except` block. It'll only ever reach here if the exception was not raised.

Reply:
Fixed.